packages/buffetjs-core/src/components/Button/index.js

```diff
@@ -7,16 +7,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { omit } from 'lodash';
-import { Button as StyledButton } from '@buffetjs/styles';
+import { Button as StyledButton, buttonsColors } from '@buffetjs/styles';
 import { Plus } from '@buffetjs/icons';
 import PrefixIcon from '../PrefixIcon';
 
 function Button(props) {
   const content = props.label ? props.label : props.children;
-  const { disabled } = props;
+  const { disabled, color } = props;
```

Comment:
You can destructure all the props object directly in the function definition `function Button({disabled, color, children, label})`.